build.sbt

```diff
@@ -65,6 +67,7 @@ lazy val mavenPlugin = Project(
   .settings(Seq(
     publishMavenStyle := true,
     crossPaths := false,
+    crossScalaVersions := Seq(scala212),
```

Comment:
Different style here vs above with removing 2.13 vs replacing with a list with only 2.12

Reply:
good point, changing to `:= Seq(scala212)` for all of those since they're only used at the 'plugin side'